tests/test_nn/test_representation.py

```diff
@@ -303,7 +318,7 @@ class WikidataTextRepresentationTests(cases.RepresentationTestCase):
 
     cls = pykeen.nn.representation.WikidataTextRepresentation
     kwargs = dict(
-        identifiers=["Q100", "Q1000"],
+        labels=["Q100", "Q1000"],
```

Comment:
This doesn't make sense. These are identifiers and labels should be looked up based on them